docker/docker-compose.ci.yml

```diff
@@ -3,7 +3,8 @@
 
 services:
   www:
-    build: ../components/proxy
+    build:
+      context: ../components/proxy
```

Comment:
Is this a style issue or does it do different things? In any case, shouldn't we make the same change for the other services?

Reply:
Ah! This is an unneeded change, the default option is `context`. I had added a build argument, which required the separation into a dict with options